src/Microsoft.AspNet.Mvc.Core/DefaultControllerActionArgumentBinder.cs

```diff
@@ -84,10 +89,13 @@ private void UpdateParameterMetadata(ModelMetadata metadata, IBinderMetadata bin
                 ValueProvider = bindingContext.ValueProvider,
             };
 
+            var modelState = actionContext.ModelState;
+            modelState.MaxAllowedErrors = _options.MaxModelValidationErrors;
```

Comment:
I understand the place `ViewDataDictionary` constructors have in this.  but it would be nice if `MaxModelValidationErrors` were set in the _one_ place `ModelStateDictionary` instances were created.  pity.